Update benefit selection language and remove requirement

raquelromano-gov opened the pull request:
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1159 and https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1162

Commit:
Fix language on contact info and benefit selection & don't require benefit selection

Commit:
Change VA to we

Commit:
Only capitalize first word of title

jbalboni commented:
@raquelromano-gov We normally include a link to the original issue and a screenshot (if the changes are visible in the UI) in the description.

raquelromano-gov commented:
Addresses issues:
-  https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1162
-  https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1159

![image](https://cloud.githubusercontent.com/assets/25439097/22916595/35ae2948-f234-11e6-96a3-ba3e12a59e71.png)

Confirmation/review page:

![image](https://cloud.githubusercontent.com/assets/25439097/22916564/1a41f716-f234-11e6-9c2e-520668f1c662.png)

![image](https://cloud.githubusercontent.com/assets/25439097/22916525/d9c52212-f233-11e6-94e1-8a2735d1604c.png)

jbalboni reviewed (approved):
Looks good to me. Added a couple comments below. Feel free to squash merge when those are addressed.

jbalboni commented on src/js/edu-benefits/1995/config/form.js:
```diff
@@ -108,18 +108,17 @@ const formConfig = {
         benefitSelection: {
-          title: 'Benefit selection',
+          title: 'Education benefit',
           path: 'benefits-eligibility/benefit-selection',
```
Missed this before. We could change the url here, since this doesn't make that much sense now. Maybe just `benefit` or `education-benefit`?

jbalboni commented on src/js/edu-benefits/1995/config/form.js:
```diff
@@ -108,18 +108,17 @@ const formConfig = {
       title: 'Benefit Selection',
```
Should change this title, too, I think.

Commit:
Change URL and page title
